support/github-to-jira-synchronization/clay.md

```diff
@@ -0,0 +1,257 @@
+# liferay/clay GitHub to JIRA syncrhonization
+
+This document describes the workflow between the [Clay](https://github.com/liferay/clay) project in GitHub and issues.liferay.com.
+
+> ⚠️ Issues are unidirectionally synced **from GitHub to JIRA.**
+
+## GitHub Setup
+
+Since the setup is mostly unidirectional, the GitHub side is pretty simple and basically just prepares an event payload every time an action is triggered in the repo.
+
+1. To manage the clay connection, [locate the Clay Exalate remote](./README.md) and navigate to it.
+2. Navigate to `Connections` > `clay-github-to-jira` > `Edit Connection`
+
+![Edit Clay Connection](./images/gh_clay_edit_connection.png)
+
+3. Verify or Update Triggers setup. At the moment, only one trigger is necessary: `is:issue`
+
+![Edit Clay Trigger](./images/gh_clay_edit_trigger.png)
```

Comment:
This link is broken. I think the right image is jira_clay_edit_trigger.png

Reply:
Mmm... I think the name is correct, since it's in the GH Setup portion, so the snapshot should belong to the `gh_` series. I think I just deleted it by mistake 🤔

Reply:
Thanks! edited the message before seeing your answer 🤦